analyzer/src/config/logger.py

```diff
@@ -21,6 +24,12 @@ def get_logger():
     if logger is not None:
         return logger
 
+    # When we are doing tests
+    if TEST:
```

Comment:
actually it could be cool to be able to define the logging level directly by setting an env var don't you think ? like LOG_LEVEL=debug
it can be for later though, as you prefer

Reply:
It's already done with LOGGING_LEVEL 🙂